Libraries/Components/DatePickerWindows/DatePickerWindows.windows.js

```diff
@@ -0,0 +1,157 @@
+         * date and time.
+         */
+        onChange: React.PropTypes.func,
+
+        /**
+         * Maximum year.
+         *
+         * Restricts the range with an upper bound on the year.
+         * Works only in 'date' and 'datetime' modes.
+         */
+        maxYear: React.PropTypes.instanceOf(Date),
+
+        /**
+         * Minimum year.
+         *
+         * Restricts the range with an lower bound on the year.
+         * Works only in 'date' and 'datetime' modes.
+         */
+        minYear: React.PropTypes.instanceOf(Date),
+
+        /**
+         * Minimum date.
+         * 
+         * Restricts the range with an lower bound.
+         * Works only in the 'calendar' mode.
+         */
+        minDate: React.PropTypes.instanceOf(Date),
+
+        /**
+         * Maximum date.
+         * 
+         * Restricts the range with an upper bound.
+         * Works only in the 'calendar' mode.
+         */
+        maxDate: React.PropTypes.instanceOf(Date),
+
+        /**
+         * The date picker mode.
+         * 
+         * Modes:
+         * - time: Time selector
+         * - date: Date selector, in a scrollable interface
+         * - datetime: date and time modes combined
+         * - calendar: Date selector using a calendar view
+         */
+        mode: React.PropTypes.oneOf(['date', 'time', 'datetime', 'calendar']),
+
+        /**
+         * Placeholder text.
+         * 
+         * Works only in 'calendar' mode.
+         */
+        placeholderText: React.PropTypes.string,
+
+        /**
+         * Calendar type, defined as a string.
+         * 
+         * Only works in 'calendar' mode. The supported values are defined in the Windows.Globalization.CalendarIdentifiers class.
+         * Some values require API v10.0.14393.0 or above
+         */
+        calendarIdentifier: React.PropTypes.oneOf('ChineseLunarCalendar', 'GregorianCalendar', 'HebrewCalendar',
+                                                  'HijriCalendar', 'JapaneseCalendar', 'JapaneseLunarCalendar',
+                                                  'JulianCalendar', 'KoreanCalendar', 'KoreanLunarCalendar',
+                                                  'PersianCalendar', 'TaiwanCalendar', 'TaiwanLunarCalendar',
+                                                  'ThaiCalendar', 'UmAlQuraCalendar', 'VietnameseLunarCalendar'),
+
+        /**
+         * First day of the week, defined as an integer with 0 being Sunday, 1 Monday, etc.
+         * Only works in calendar mode.
+         */
+        firstDayOfWeek: React.PropTypes.oneOf(0, 1, 2, 3, 4, 5, 6)
+    },
+    getDefaultProps: function() {
+      return {
+        mode: 'datetime',
+        firstDayOfWeek: 0,
+        calendarIdentifier: 'GregorianCalendar',
+      };
+    },
+    _onDateChange: function(event) {
+      this.props.onDateChange && this.props.onDateChange(new Date(event.nativeEvent.date));
+    },
+    render: function() {
```

Comment:
I am inclined to agree. The sole reason for that decision was consistency with the iOS and Android counterparts. Will do.